config/jobs/kubernetes/sig-cloud-provider/azure/OWNERS

```diff
@@ -2,19 +2,21 @@
 
 reviewers:
```

Comment:
let's add @Jont828 and @jsturtevant as well?

Reply:
I'll open a follow-up PR for this later today